index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
```

Comment:
```suggestion
The <a>verifiable credential</a> and <a>verifiable presentation</a> data models leverage a
```

All special terms defined in this document need to be linked.